pulsar-broker/src/main/java/org/apache/pulsar/broker/loadbalance/extensions/scheduler/TransferShedder.java

```diff
@@ -671,6 +685,18 @@ private boolean isUnderLoaded(LoadManagerContext context, String broker, double
                 context.brokerConfiguration().getLoadBalancerBrokerLoadTargetStd() / 2));
     }
 
+    private boolean isOverLoaded(LoadManagerContext context, String broker, double avgLoad) {
+        var brokerLoadDataOptional = context.brokerLoadDataStore().get(broker);
+        if (brokerLoadDataOptional.isEmpty()) {
+            return false;
+        }
+        var overloadThreshold = context.brokerConfiguration()
+                .getLoadBalancerBrokerOverloadedThresholdPercentage() / 100.0;
+        var brokerLoadData = brokerLoadDataOptional.get();
+        var load = brokerLoadData.getWeightedMaxEMA();
+        return load > overloadThreshold && load > avgLoad * 2;
```

Comment:
Could you explain why the load needs to be greater than twice `avgLoad`?

Reply:
if all of the brokers are overloaded, then we don't want to unload.

[100, 100, 100, 100, 100], avg = 100 => no unload

[1, 1, 1, 1, 100] avg = 20.8 => unload because 100 > 85(threshold) && 100 > 20.8 *2.

Reply:
The multiplier 2 is somewhat arbitrary, and I think two is good to be conservative.

Reply:
@gaoran10 In fact, I think we can use the target std for this comparison as well. `load > avgLoad + targetStd` 
Updated. PTAL.